bitnami/redis/Chart.yaml

```diff
@@ -19,4 +19,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 12.2.4
+version: 12.2.5
```

Comment:
As it is adding a feature, could you bump the minor version instead?

Reply:
Thanks @javsalgar ! done